spec/rspec/core/ordering_spec.rb

```diff
@@ -81,6 +81,23 @@ def order_with(seed)
         end
       end
 
+      RSpec.describe ModificationTime do
+        require 'time'
```

Comment:
Is this required for `File.mtime` as that could be problematic

Reply:
It is necessary. I wasn't sure about where I should require it.

Reply:
You will need to require it in the test in a subprocess (we have a helper for that) to avoid contaminating the other specs, you will also need to check that it has been required and error if not, we don't want to automatically require it because it produces false positives.

Reply:
Is there any other spec doing this, so I can look it up? What helper is that?

Reply:
I may be wrong but Jon probably speak about https://github.com/rspec/rspec-support/blob/main/lib/rspec/support/spec/in_sub_process.rb and an example in rspec-core https://github.com/rspec/rspec-core/blob/5d87c28bb828ba14b54a0d8cfddf044894009314/spec/rspec/core/bisect/utilities_spec.rb#L28-L33

Reply:
I figured out how to create a Time object without parsing a string (which was what made me `require `'time'`).